media-libs/kvazaar/kvazaar-9999.ebuild

```diff
@@ -24,9 +24,13 @@ SLOT="0/4"
 IUSE="static-libs test"
 REQUIRED_USE="test? ( static-libs )"
 
+RESTRICT="!test? ( test )"
```

Comment:
what's this ? I seem to recall some logic behind that but RESTRICT="test" seems simpler